src/components/login-form.js

```diff
@@ -1,14 +1,110 @@
-import React from "react";
-import styled from "styled-components";
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+
+import VoteForm from './vote-form';
 
 export default function LoginForm() {
-  return <Wrapper>안녕 나는 로그인 폼!</Wrapper>;
+	const [logForm, setLogForm] = useState({ email: '', password: '' });
+	const [isloged, setloged] = useState(false);
+
+	const erasePW = () => {
+		document.getElementById('password').value = '';
+	};
+
+	const eraseEmail = () => {
+		document.getElementById('email').value = '';
+	};
```

Comment:
1. 여기서 2중 화살표 함수를 사용할 수 있겠네요 ㅎ
호출시엔 다음과 같이 사용하면 됩니다.
```
erase('email')();
erase('password')();
```
2. dom에 직접 접근하기보단 react state를 사용해주세요!

```suggestion
	const erase = (name) => () => {
	  setLoginForm({
	    ...loginForm,
	    [name] : ''
	  })	
	};
```